app/controllers/ajax/moderation_controller.rb

```diff
@@ -164,9 +164,9 @@ def privilege
     target_user = User.find_by_screen_name(params[:user])
 
     @message = I18n.t('messages.moderation.privilege.nope')
-    return unless %w(blogger supporter moderator admin contributor translator).include? params[:type].downcase
+    return unless %w(moderator admin).include? params[:type].downcase
 
-    if %w(supporter moderator admin).include?(params[:type].downcase) && !current_user.has_role?(:administrator)
+    if %w(moderator admin).include?(params[:type].downcase) && !current_user.has_role?(:administrator)
```

Comment:
the first part of this `if` can never be false (already checked by the `return unless ...` above) --> remove it